mkdocs.yml

```diff
@@ -3,15 +3,27 @@ site_url: https://HuMoN-Research-Lab.github.io
 
 theme:
   name: material
-#  logo: assets/humon_sassy_skeleton_logo.png
+  logo: assets/skelly_freemocap_logo.png
+  favicon: assets/skelly_freemocap_favicon.ico
+  features: 
+    - navigation.tabs
+  palette:
+    scheme: freemocap
+extra_css:
+  - stylesheets/extra.css
   
-  markdown_extensions:
+markdown_extensions:
   - abbr
   - attr_list
+  - md_in_html
   - pymdownx.snippets:
       auto_append:
         - includes/abbreviations.md
 
-nav:
-    - Home: index.md
-    - How to update these docs: how-to-update-these-docs.md
+plugins:
+  - glightbox
```

Comment:
what does `glightbox` do?

Reply:
Copy that Jon, I'll make that adjustment.